docs/BUILD.bazel

```diff
@@ -235,16 +235,15 @@ genrule(
             sass:da_theme/static/css
         ../$(location :grunt) build
 
-        tar \
+        tar c da_theme \
             --owner=1000 \
             --group=1000 \
-            --mtime=1337 \
+            --mtime=2000-01-01\ 00:00Z \
```

Comment:
`--mtime=1337` did only affect the time of day but not the date, also it was sensitive to timezone.